spec/rspec_otel/rspec_otel_spec.rb

```diff
@@ -1,21 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I didn't see this file the first time I wrote my tests, I moved these specs into the other RspecOtel spec file which I think is in the more expected location.